.pylintrc

```diff
@@ -486,10 +487,3 @@ max-statements=50
 
 # Minimum number of public methods for a class (see R0903).
 min-public-methods=2
-
-
-[EXCEPTIONS]
-
-# Exceptions that will emit a warning when being caught. Defaults to
-# "Exception".
-overgeneral-exceptions=Exception
```

Comment:
Before giving up and remove the warning I would try with builtins.Exception, I think this warning is useful

Reply:
All right, changed it.